command/pr.go

```diff
@@ -471,41 +482,146 @@ func prMerge(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	if pr.State == "MERGED" {
+	if pr.Mergeable == "CONFLICTING" {
+		err := fmt.Errorf("%s Pull request #%d has conflicts and isn't mergeable ", utils.Red("!"), pr.Number)
+		return err
+	} else if pr.Mergeable == "UNKNOWN" {
+		err := fmt.Errorf("%s Pull request #%d can't be merged right now; try again in a few seconds", utils.Red("!"), pr.Number)
```

Comment:
"UNKNOWN" usually means that mergeability is currently still being calculated. Instead of presenting this as an error, we could be optimistic and assume that the PR is OK to merge and allow the API merge operation to proceed—it has a chance of succeeding, and if it fails the API error message should let us know why.